crates/polars-io/src/parquet/read_impl.rs

```diff
@@ -99,9 +99,11 @@ pub(super) fn array_iter_to_series(
 fn materialize_hive_partitions(
     df: &mut DataFrame,
     hive_partition_columns: Option<&[Series]>,
-    num_rows: usize,
+    num_rows: Option<usize>,
```

Comment:
Where is this `Some` passed? I only see it called with `None`.

Reply:
Hmm, nowhere actually - will remove the parameter.